src/main/java/com/scrive/pdftools/tools/utils/Resources.java

```diff
@@ -0,0 +1,10 @@
+package com.scrive.pdftools.tools.utils;
+
+import java.io.InputStream;
+
+public class Resources {
+    public static InputStream getResource(String res) {
```

Comment:
`IOUtils.resourceToByteArray` ? https://commons.apache.org/proper/commons-io/javadocs/api-release/org/apache/commons/io/IOUtils.html#resourceToByteArray-java.lang.String-

Reply:
I removed this class. It's not needed because we don't need to load any resources.